src/tests/tablet/schedule/AMeetingCannotBeCreatedWithoutSubject.java

```diff
@@ -0,0 +1,43 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC09: Verify a meeting cannot be created if the subject is not specified
+ * @author Asael Calizaya
+ *
+ */
+public class AMeetingCannotBeCreatedWithoutSubject {
+
+	@Test(groups = "ACCEPTANCE")
+	public void testAMeetingCannotBeCreatedWithoutSubject() {
+		ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+		List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");
+		String organizer = meetingData.get(0).get("Organizer");
+		String startTime = meetingData.get(0).get("Start time");
+		String endTime = meetingData.get(0).get("End time");
+		String attendee = meetingData.get(0).get("Attendee");
+		String body = meetingData.get(0).get("Body");
+
+		HomeTabletPage homeTabletPage = new HomeTabletPage();
+		SchedulePage schedulePage = homeTabletPage.clickScheduleBtn();
+		schedulePage.setOrganizerTxtBox(organizer)
+		.setStartTimeDate(startTime)
```

Comment:
Please try to omitted the steps that are not necessary...review this in of all your TCs